src/components/Tokens/TokenDetails/StatsSection.tsx

```diff
@@ -77,44 +83,100 @@ function Stat({
 type StatsSectionProps = {
   chainId: ChainId
   address: string
-  priceLow52W?: NumericStat
-  priceHigh52W?: NumericStat
-  TVL?: NumericStat
-  volume24H?: NumericStat
+  tokenQueryData: TokenQueryData
 }
 export default function StatsSection(props: StatsSectionProps) {
-  const { chainId, address, priceLow52W, priceHigh52W, TVL, volume24H } = props
+  const { chainId, address, tokenQueryData } = props
   const { label, infoLink } = getChainInfo(chainId)
+  const isInfoTDPEnabled = useInfoTDPEnabled()
+
+  const tokenMarketInfo = tokenQueryData?.market
+  const tokenProjectMarketInfo = tokenQueryData?.project?.markets?.[0] // aggregated market price from CoinGecko
+
+  const FDV = tokenProjectMarketInfo?.fullyDilutedValuation?.value
+  const marketCap = tokenProjectMarketInfo?.marketCap?.value
```

Comment:
more of a BE Q but could FDV and MarketCap not be queried from `tokenQueryData.market` instead of the token project's market? It's a little confusing having the two different marketInfo objects imo

Reply:
definitely agree that it is confusing - tokenProjectMarket & tokenMarket are two different things. 

https://github.com/Uniswap/interface/blob/22be28bcb2c96d764e64f4ea3cfb5301aea1bfc1/src/graphql/data/Token.ts#L11-13